### ISSUE TITLE
youtube-dl taking long time to download audio

### ISSUE THREAD

**Issue by GarRudnyai** [ytdl-org/youtube-dl#30262]
## Checklist

- [ ] I'm reporting a broken site support issue
- [ x] I've verified that I'm running youtube-dl version **2021.06.06**
- [ x] I've checked that all provided URLs are alive and playable in a browser
- [ x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ x] I've searched the bugtracker for similar bug reports including closed ones
- [ x] I've read bugs section in FAQ

## Verbose log

```
[youtube] y97gX9QSwZk: Downloading webpage
[download] Destination: y97gX9QSwZk.wav
[download] 100% of 4.47MiB in 01:13
[ffmpeg] Post-process file y97gX9QSwZk.wav exists, skipping
```

## Code (embedded in python)
```
video_id = get_yt_video_id(url) # this method works; returns a youtube video id given its url
    ydl_opts = {
        'format': 'bestaudio/best',
        'nocheckcertificate': True,
        'outtmpl': (video_id + ".wav"),
        'postprocessors': [{
            'key': 'FFmpegExtractAudio',
            'preferredcodec': 'wav',
            'preferredquality': '192',
        }],
    }
    with youtube_dl.YoutubeDL(ydl_opts) as ydl:
        info_dict = ydl.extract_info(url, download=True)
```

## Description

I am trying to just download the audio from a YouTube video using youtube-dl, but it is taking me as long as one to two minutes while it should be no more than 20 seconds for short videos (under five mins). I am running the code above in a heroku app, and when I try to download youtube videos to the server in production the app almost always times out (heroku waits 55 seconds before timing out). I only want to download the audio, so if my code downloads the video as well, please let me know how I can download only audio. Besides that, why are my speeds so slow and how can I get them to more reasonable levels? Thanks!

**Comment by dirkf** [comment]
Duplicate of #29326: [TL;DR](https://github.com/ytdl-org/youtube-dl/issues/29326#issuecomment-974052706).

**Comment by GarRudnyai** [comment]
Thank you, @dirkf ! Appreciate the help! Just have one question though- what is the "PR" that you and other contributors reference in these issues?

**Comment by dirkf** [comment]
Closed in af9e72507.
